woocommerce-gateway-paylike/assets/js/paylike_checkout.js

```diff
@@ -134,6 +134,7 @@ jQuery(function ($) {
                 return wc_paylike_form.escapeQoutes(phone);
             },
             onSubmit: function (e) {
+                if (!wc_paylike_form.validateShipmondo()) return true;

```

Comment:
Looking at this, I think this should return false, and be in the `isPaylikeModalNeeded` method, doesn't that work?

Reply:
If i move it to the isPaylikeModalNeeded then i would have to return false.
But you asked me to put it in the onSubmit function. :-)
> and only use one line in the onSubmit function

Reply:
I see, sorry about that, can you please put it back, don't know where I was thinking.

Reply:
Ok, so is this all you need? If so, I can merge it, should make it to the WordPress plugin after I run some tests.

Reply:
Yes, that's all, you are free to merge.